### DISCUSSION TITLE
Why are external DAC's "L/R LINEOUT" recommended load resistances so high?

### DISCUSSION THREAD

**Discussion by mrhempman69** [pschatzmann/ESP32-A2DP#489] [Q&A] [answered]
Hi, I'm the author of https://github.com/pschatzmann/ESP32-A2DP/discussions/482. Trying to debug why I still can't hear audio, even when using a proper MCLK. Trying to determine if it's a problem with the library, or elsewhere... It's a bit hard with cheap Amazon oscilloscopes though@@

My current concern is, that the [datasheet](https://www.nuvoton.com/export/resource-files/DS_NAU8402_DataSheet_EN_Rev1.7.pdf) for the NAU8402 says on page 8 that "typical load resistance" is 1 kiloOhm to 47 kiloOhm for the "analog out" pins AKA speaker out pins... Presumably this is the ohmmage of the speakers that you would connect to the chip.

### **PCM5102a:**
![image](https://github.com/pschatzmann/ESP32-A2DP/assets/41412018/f96be9a6-1baf-40ec-b253-0d7058036aaf)

Furthermore, the PCM5102a (which is used on a breakout board in the wiki) also has a recommended minimum load resistance of 1 kiloOhm, with a nominal one of 10 kiloOhm. (These values are from the PCM5102a's datasheet, so you'd just be using the chip by itself for these to be relevant, not its more complete breakout board in the wiki.) Still, even on the breakout board, there are only 460 Ohm resistors leading to the TRS jack (similar to its typical application diagram in the datasheet). So it would seem the PCM5102a breakout board actually doesn't reach its suggested load resistance when using 8 to 64 Ohm speakers (most speakers are in this range)...

### **NAU8402:**
![image](https://github.com/pschatzmann/ESP32-A2DP/assets/41412018/a83b0de9-33e9-49a4-86c8-fe0f9e27c218)

In the "typical application" circuit diagram for the NAU8402 on page 14, it even says "...the 0dBFs level at the L & R AUDIO OUT terminals will be 2 Vrms, when a **10-kΏ** load is applied...". It's peculiar that it recommends 500 Ohm resistors before reaching the "AUDIO OUT" labels, as if the remaining 9.5 kiloOhm resistance must come from whatever "AUDIO OUT" is supposed to be...

With the NAU8402 I get no sound, when using a 32 Ohm speaker, while the PCM5102 does output some sound, albeit noisy. 

My question is, what is going on with these recommended load resistances? **Typical speaker values range from 8 Ohm to 64 Ohm.** What does it mean when it suggests having load resistances in the **kiloOhm** range? How do you deal with this from a circuit design standpoint?

If it's not obvious, **I can't find any speakers in the kiloOhm range** to even test with.

**Comment by pschatzmann** [comment, marked as answer]
I don't really know much about electronics, but here is my understanding: You can't drive any speakers from the output of a DAC chip, but it is usually good enough for some earphones.

To drive some speakers, you will need an amplifier! 
If you check out the spec of the AUX input you will find your 10kOhm...

**Reply by mrhempman69** [reply]
The Great Maintainer O Wise one Saves the Day

On the breakout board, there is 15 kiloOhm on the AUX jack like you said, but not on my personal TRS jacks! Still can't believe any of that... I did finally get (some very poor quality) audio with a 15 kiloOhm resistor added! Took days on this LOL... Thought I tried this already btw, but with a lower value resistor, I guess is what I did... Audio quality is not a concern, I have to undo all the settings changes that I made and get it to work, also look into an amplifier like you said, however, I will only be driving earphones!

**Comment by pschatzmann** [comment]
This might be of interrest as well: https://en.wikipedia.org/wiki/Line_level
